VideoApp/VideoApp/Stores/UserActivity/UserActivityStore.swift

```diff
@@ -28,9 +29,20 @@ class UserActivityStore: UserActivityStoreWriting {
     }
     
     @discardableResult func continueUserActivity(_ userActivity: NSUserActivity) -> Bool {
-        guard let url = userActivity.webpageURL, let deepLink = DeepLink(url: url) else { return false }
-        
-        deepLinkStore.cache(deepLink: deepLink)
+        if let startCallHandle = userActivity.startCallHandle {
+            deepLinkStore.cache(deepLink: .room(roomName: startCallHandle))
+        } else if let url = userActivity.webpageURL, let deepLink = DeepLink(url: url) {
+            deepLinkStore.cache(deepLink: deepLink)
+        } else {
+            return false
+        }
+
         return true
     }
 }
+
+private extension NSUserActivity {
+    var startCallHandle: String? {
+        (interaction?.intent as? INStartVideoCallIntent)?.contacts?.first?.personHandle?.value
```

Comment:
@bobiechen-twilio I'm getting a warning that `INStartVideoCallIntent` is deprecated. It says to use `INStartCallIntent` instead but it wasn't working and it doesn't seem well documented. Wondering if you are familiar with this?